docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md

```diff
@@ -0,0 +1,37 @@
+---
+slug: how-to-create-find-organize-projects
+title: How to Create, Find, and Organize Projects
+tags: []
+toc_max_heading_level: 5
+---
+# How to Create a Project
```

Comment:
Only the title of the page should be H1, otherwise it doesn't show up in the TOC or headings overview on the right. Can you make this H2 (##)? And all other headings one level below?

Reply:
Got it